iphone/Classes/TiUIApplicationShortcutsProxy.m

```diff
@@ -166,12 +166,6 @@ - (void)addDynamicShortcut:(id)args
     return;
   }
 
-  if ([self typeExists:[args objectForKey:@"identifier"]]) {
-    NSLog(@"[ERROR] Ti.UI.ApplicationShortcuts: The identifier for the shortcut %@ already exists. This field must be unique.",
-        [args objectForKey:@"identifier"]);
-    return;
-  }
-
   UIApplicationShortcutItem *shortcut = [[[UIApplicationShortcutItem alloc] initWithType:[args objectForKey:@"identifier"]
```

Comment:
Does it make sense to rename this to `itemtype` too? Not sure why it's named `identifier` here and `itemtype` in the other place.

Reply:
SDK 7.5.0 renamed it to `identifier` for the new API on the Ti.UI namespace which uses `identifier` for better parity with Android. I don't exactly remember why the old one wasn't just used here as well.